### PR TITLE
Fix travis-ci tests

### PR DESCRIPTION
**Pull request by rendhalver**
It looks like the tests haven't been run for a while and are failing in #18 
This is an attempt to get those working again.

### CODE REVIEW

File: .fixtures.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 fixtures:
-  repositories:
+  forge_modules:
     stdlib:
-      repo: 'git://github.com/puppetlabs/puppetlabs-stdlib.git'
```
Comment (ghoneycutt) [inline .fixtures.yml]: suggest using https with github as its more likely allowed behind corp firewalls

```yaml
fixtures:
  repositories:
    stdlib:
      repo: 'https://github.com/puppetlabs/puppetlabs-stdlib.git'
      ref: '4.6.0'
  symlinks:
    cgroups: "#{source_dir}"
```

File: metadata.json [hunk 2]
```diff
@@ -31,10 +31,6 @@
   ],
   "requirements": [
     {
-      "name": "pe",
```
Comment (ghoneycutt) [inline metadata.json]: good catch!

File: .fixtures.yml (hunk 1 above)
Comment (rendhalver) [reply .fixtures.yml]: I find using forge links a bit easier especially when tracking particular versions.

**Comment by Phil-Friderici** [comment]
Thanks guys !
I have fixed the spec tests in v2.1.2. So I'm closing this. Please re-open if still needed.

**Pull request closed without merging**